octoprint_findmymrbeam/__init__.py

```diff
@@ -61,12 +60,45 @@ def __init__(self):
 		self._all_secrets = self._not_so_secret + [self._secret]
 
 	def initialize(self):
+		self._uuid = self._get_setting([["plugins", "discovery", "upnpUuid"], ], ["public", "uuid"])
+		if self._uuid is None:
+			self._generate_uuid()
+		self._search_id = self._settings.get(["public", "search_id"])
+		if self._search_id is None:
+			self._generate_search_id()
```

Comment:
```
self._uuid = ... or ...
self._search_id = ... or ...
```
You should make the generator functions return the value instead of assigning it to the class attribute